libs/newsletters-data-client/src/lib/draft-to-newsletter.ts

```diff
@@ -28,36 +32,38 @@ export const withDefaultNewsletterValuesAndDerivedFields = (
 ): DraftNewsletterData &
 	Pick<NewsletterData, NewsletterFieldsDerivedFromName> => {
 	const derivedFields = deriveNewsletterFieldsFromName(draft.name ?? '');
-
-	if (draft.renderingOptions) {
-		return {
-			...defaultNewsletterValues,
-			...derivedFields,
-			...draft,
-			//prevent an explicit undefined status on the draft overriding the default
-			status: draft.status ? draft.status : defaultNewsletterValues.status,
-			renderingOptions: {
+	//prevent an explicit undefined status on the draft overriding the default
+	const getDefaultedStatus = () =>
```

Comment:
does this need to be a function? I may be missing something but this looks the same as:

```
draft.status ?? defaultNewsletterValues.status
```

We could inline that rather than declaring a function right

Reply:
👍  done